frontend/js/views/list.js

```diff
@@ -243,90 +239,104 @@ define(["underscore",
             },
 
             /**
-             * Select the given annotation
-             * @alias module:views-list.List#select
-             * @param  {Annotation} annotations The annotation to select
+             * Update the annotation views to reflect the current selection
+             * @alias module:views-list.List#renderSelection
+             * @param {Annotation?} selection the currently selected annotation
+             * @param {Annotation?} previousSelection the previously selected annotation
              */
-            select: function (annotations) {
-                var annotation,
-                    i,
-                    view,
-                    selectedAnnotations = [];
-
-                // only remove the annotations
-                for (i = 0; i < annotations.length; i++) {
-                    annotation = annotations[i];
-                    if (annotation) {
-                        view = this.getViewFromAnnotation(annotation.get("id"));
-
-                        // If view not found, annotation has been newly created
-                        if (!_.isUndefined(view)) {
-                            selectedAnnotations[i] = view;
-                        }
-                    }
+            renderSelection: function (selection, previousSelection) {
+                if (previousSelection) {
+                    this.getViewFromAnnotation(previousSelection.id)
+                        .$el.removeClass("selected");
                 }
+                if (selection) {
+                    var view = this.getViewFromAnnotation(selection.id).$el;
```

Comment:
Creating a freetext annotation produces a type error: 
`TypeError: this.getViewFromAnnotation(...) is undefined`
Doesn't seem to hamper functionality, but may be worth fixing nonetheless.

Reply:
Actually this was exactly the cause for #356. Should be fixed by 2c75976 now.